drivers/bluetooth/hci/spi.c

```diff
@@ -190,13 +190,13 @@ static int configure_cs(void)
 
 static void kick_cs(void)
 {
-	gpio_pin_set_dt(&bus.config.cs->gpio, 1);
-	gpio_pin_set_dt(&bus.config.cs->gpio, 0);
+	gpio_pin_set_dt(&bus.config.cs->gpio, false);
+	gpio_pin_set_dt(&bus.config.cs->gpio, true);
```

Comment:
gpio api defines 2nd argument as 'int', so 0/1 is fine as per API. If there's an actual issue, it should also be fixed at api level (and extended to users). I'd suggest to raise a dedicated issue for this point.

Reply:
Ok, right, but I don't want to change API, so let's keep integer 0/1, and change values to have logical value instead of pin value.